README.md

```diff
@@ -14,7 +14,6 @@ nuget restore packages.config -PackagesDirectory packages
 - Run tests with your favorite test runner
 
 ## Code Status
-[![NuGet version](https://img.shields.io/nuget/v/OSPSuite.FuncParser.svg?style=flat)](https://www.nuget.org/packages/OSPSuite.FuncParser)
```

Comment:
This is more of a proposal I guess. Do we want to keep this badge? I think we haven't been pushing to nuget.org

Reply:
I think, we don't need it. (Already created an issue for this: https://github.com/Open-Systems-Pharmacology/OSPSuite.FuncParser/issues/64)